CI/physmon/phys_perf_mon.sh

```diff
@@ -247,7 +247,10 @@ if [[ "$mode" == "all" || "$mode" == "simulation" ]]; then
     simulation geant4
 fi
 
-CI/physmon/summary.py $outdir/*.html $outdir/summary.html
+CI/physmon/summary.py $outdir/*.html summary.md
```

Comment:
maybe the python script could output both `md` and `html` just to remove a bit of logic from bash

Reply:
yeah I think we should preserve single source. so in python we could copy paste to the other output and prepend the html stuff